src/main/java/org/sonar/plugins/stash/StashPluginConfiguration.java

```diff
@@ -114,6 +114,10 @@ public boolean includeExistingIssues() {
     return settings.getBoolean(StashPlugin.STASH_INCLUDE_EXISTING_ISSUES);
   }
 
+  public int includeFilesInOverview() {

```

Comment:
I would prefer a slightly different name `includedFilesInOverview`. The current name sounds like a boolean.
Would also affect various variable names.